test/e2e/framework/pod/resize.go

```diff
@@ -24,6 +24,7 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/opencontainers/cgroups"
```

Comment:
this package only builds on linux, so I recommend putting it in a separate _linux.go until we can sort out the fix

Reply:
(it should fail presubmits until then)